Update DD documents to reflect new design (#64)
uc-cdis/gen3.org: content/resources/user/dictionary.md

* * *

### Viewing Data Dictionary as a Graph:
* View the Data Dictionary as a graph to see each node, its properties, and its relationships.

> * Relationships between nodes are represented by arrows from one node to another..

Reviewer: ```suggestion
* Relationships between nodes are represented by arrows from one node to another.
```

![Dictionary Graph](graph.png)

### Viewing Data Dictionary as Tables:

> * View the Data Dictionary as a table to see the name and a brief description of each node, organized by node type.

Reviewer: ```suggestion
* View the Data Dictionary as a table to see the name and a brief description of each node, organized by node category.
```

### Toggling Between Different Views
* Click on "Graph View" or "Table View" in the top left of the Dictionary Viewer to switch between these two views.
![Data Dictionary Viewer](switch-to-table.gif)

### Viewing Properties of a Single Node:
* The Properties View of a node lists each property, the property's type, whether or not the property is required, and a brief description of the property.
![Properties of a Node](properties.png)

### Opening Properties View:
* Click on a node to see its parent nodes and information about its properties.
![Opening Properties View](properties-view.gif)

### Search for Node, Property, or Description:
* Enter a search a term in the search box.
* Search terms can be node names, node descriptions, property names, and property descriptions.
* While typing, a list of suggestions appears below the search bar. Click on a suggestion to search for it.
* A search will display all nodes that contain the search term in either its name or description, or its properties' names or descriptions.
* A history of your searches appears below the search bar. Click on an item here to display the results again.
![Search](search.gif)

### Download Submission Templates:

> * You can download a submission template to help you with submitting files.

Reviewer: ```suggestion
* In the Graph View, you can download submission templates to help you with submitting your project's data.
```

> * Submission templates can be either JSON or TSV files.

Reviewer: ```suggestion
* Click a node in the graph that you'd like to submit, and all parent nodes will be highlighted.
* Click the "Download templates" button to the left of the highlighted graph. Submission templates can be either JSON or TSV files.
```

![Properties of a Node](properties.png)

### Opening Properties View:
* Click on a node to see its parent nodes and information about its properties.
![Opening Properties View](properties-view.gif)

### Search for Node, Property, or Description:
* Enter a search a term in the search box.
* Search terms can be node names, node descriptions, property names, and property descriptions.
* While typing, a list of suggestions appears below the search bar. Click on a suggestion to search for it.
* A search will display all nodes that contain the search term in either its name or description, or its properties' names or descriptions.
* A history of your searches appears below the search bar. Click on an item here to display the results again.
![Search](search.gif)

### Download Submission Templates:
* In the Graph View, you can download submission templates to help you with submitting your project's data.
* Click a node in the graph that you'd like to submit, and all parent nodes will be highlighted.
* Click the "Download templates" button to the left of the highlighted graph. Submission templates can be either JSON or TSV files.

> * Your download will be a ZIP file containing containing multiple files: One is a submission template for the selected node, and the others are submission templates for each parent node.

Reviewer: ```suggestion
* Your download will be a ZIP file containing multiple files: One is a submission template for the selected node, and the others are submission templates for each parent node.
```
